### ISSUE TITLE
Wii U networking **very** slow

### ISSUE THREAD

**Issue by fr500** [libretro/RetroArch#7279]
## Description

Networking on Wii U seems to be very slow, downloading assets takes about 15 minutes, WiFi signal is great in the room were my Wii U is located.

### Expected behavior

Downloads should be faster like on any other platform

### Steps to reproduce the bug

1. [First step]
2. [Second step]
3. [and so on...]

### Bisect Results

[Try to bisect and tell us when this started happening]

### Version/Commit
You can find this information under Information/System Information

- RetroArch: [version/commit]

### Environment information

- OS: [The operating system you're running]
- Compiler: [In case you are running local builds]

**Comment by QuarkTheAwesome** [comment]
I found some "benchmarks" from the eShop in [this YouTube video](https://www.youtube.com/watch?v=rHC3Ie5pYgI) - taking their average of 10Mb/s on WiFi we should be pulling the 100MiB assets zip in roughly 80 seconds. Clearly something is very wrong, either in RetroArch specifically or the homebrew method to access the Internet. I'd be curious to see a console routed through Wireshark or something to do some real benchmarking of the transfer rate and packet tomfoolery.

I've seen the same sort of slowness when downloading the assets zip - I didn't time it or anything, but watching the percentage tick up was so comically slow that I gave up before it hit 5%.
